build.gradle.kts

```diff
@@ -150,17 +193,22 @@ tasks.register("createRpm") {
   }
 }
 
+/*
+ * Task: createMsi
+ *
+ * Creates MSI installater file for Microsoft Windows.
+ */
 tasks.register("createMsi") {
   group = "build"
-  description = "Makes the Windows platform specific package"
+  description = "Makes the Windows platform specific package."
```

Comment:
Here, you didn't add "file" to the end. Consider making it consistent.
(The best option is to remove the redundant "file"s.)